src/components/molecules/NavItem/index.jsx

```diff
@@ -0,0 +1,3 @@
+import NavItem from './NavItem';
```

Comment:
suggestion: We can reduce the two line to:

```suggestion
export  { default } from './NavItem';
```